src/data/part.lean

```diff
@@ -200,6 +200,9 @@ dif_neg id
   get_or_else (some a) d = a :=
 dif_pos trivial
 
+lemma get_or_else_of_dom (a : part α) (h : a.dom) [decidable a.dom] (d : α) :
```

Comment:
Is it worth adding `of_not_dom` for completeness too?

Reply:
Yeah, sounds good, added.